docs/index.md

```diff
@@ -7,305 +7,46 @@ page_keywords: docker, swarm, clustering
-       agent-0: 172.31.40.100:2375
-        └ Containers: 0
-        └ Reserved CPUs: 0 / 1
-        └ Reserved Memory: 0 B / 514.5 MiB
-      ```
-
-    If you are running a test cluster without TLS enabled, you may get an error. In that case, be sure to unset `DOCKER_TLS_VERIFY` with:
-
-      ```bash
-    	$ unset DOCKER_TLS_VERIFY
-      ```
-
-## Using the docker CLI
-
-You can now use the regular Docker CLI to access your nodes:
-
-```bash
-docker -H tcp://<manager_ip:manager_port> info
-docker -H tcp://<manager_ip:manager_port> run ...
-docker -H tcp://<manager_ip:manager_port> ps
-docker -H tcp://<manager_ip:manager_port> logs ...
-```
-
-## List nodes in your cluster
-
-You can get a list of all your running nodes using the `swarm list` command:
-
-```bash
-docker run --rm swarm list token://<cluster_id>
-<node_ip:2375>
-```
-
-For example:
-
-```bash
-$ docker run --rm swarm list token://6856663cdefdec325839a4b7e1de38e8
-172.31.40.100:2375
-172.31.40.101:2375
-172.31.40.102:2375
-```
+## Understand swarm creation
 
-## TLS
+You can create a swarm on your network by pulling the Docker Swarm image. Then, using Docker, you can configure the swarm manager and all the nodes to run Docker Swarm. This method requires that you:
 
-Swarm supports TLS authentication between the CLI and Swarm but also between
-Swarm and the Docker nodes. _However_, all the Docker daemon certificates and client
-certificates **must** be signed using the same CA-certificate.
+* open identical TCP ports on every node
+* install Docker on each node
+* create and manage TLS certificates to secure your swarm
 
-In order to enable TLS for both client and server, the same command line options
-as Docker can be specified:
+This manual method is an advanced method and best suited for experienced administrators or programmers contributing to Docker Swarm. The alternative is to use `docker-machine` to install a swarm.  
 
-```bash
-swarm manage --tlsverify --tlscacert=<CACERT> --tlscert=<CERT> --tlskey=<KEY> [...]
-```
+Docker Machine is allows you to quickly install a Docker Swarm on cloud providers or inside your own data center. If you have VirtualBox installed on your local machine, you can quickly build and explore Docker Swarm in your local environment. This method automatically generates a certificate to secure your swarm.
 
-Please refer to the [Docker documentation](https://docs.docker.com/articles/https/)
-for more information on how to set up TLS authentication on Docker and generating
-the certificates.
+Using Docker Machine is the best method for users getting started with Swarm for the first time. To try the recommended method of getting started, see [Get Started with Docker Swarm](install-w-machine.md). 
 
-> **Note**: Swarm certificates must be generated with `extendedKeyUsage = clientAuth,serverAuth`.
+If you are an advanced user or interested in contributing, see [Create a swarm for development](install-manual.md).
```

Comment:
@aluzzardi  how about experienced network administrator?  This is really about guiding users.  This product is still in BETA so really no one would ever put it into production regardless. So, everyone is "trying it out" and the machine is far simpler.